From 578ba571e42fd1dfafcdbf4e65ee7fdffb681aec Mon Sep 17 00:00:00 2001
From: Eric Anholt <eric@anholt.net>
Date: Thu, 28 Jan 2010 16:13:29 -0800
Patch-mainline: 2.6.34
References: fate#310916
Git-commit: faa7bde6cb1227d13d011042f17bda0869c3bd1e
Subject: [PATCH 0459/2588] drm/i915: Correct the Sandybridge chipset info
 structs.

Disables CXSR until it's done, and sets the mobile bit on mobile.

Signed-off-by: Eric Anholt <eric@anholt.net>
(cherry picked from commit faa7bde6cb1227d13d011042f17bda0869c3bd1e)

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 drivers/gpu/drm/i915/i915_drv.c |    4 +---
 1 files changed, 1 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/i915/i915_drv.c b/drivers/gpu/drm/i915/i915_drv.c
index 5050cd3..d1b0f8f 100644
--- a/drivers/gpu/drm/i915/i915_drv.c
+++ b/drivers/gpu/drm/i915/i915_drv.c
@@ -138,13 +138,11 @@ const static struct intel_device_info intel_ironlake_m_info = {
 
 const static struct intel_device_info intel_sandybridge_d_info = {
 	.is_i965g = 1, .is_i9xx = 1, .need_gfx_hws = 1,
-	.has_pipe_cxsr = 1,
 	.has_hotplug = 1,
 };
 
 const static struct intel_device_info intel_sandybridge_m_info = {
-	.is_i965g = 1, .is_i9xx = 1, .need_gfx_hws = 1,
-	.has_pipe_cxsr = 1,
+	.is_i965g = 1, .is_mobile = 1, .is_i9xx = 1, .need_gfx_hws = 1,
 	.has_hotplug = 1,
 };
 
-- 
1.7.6

